guide-us-english.tex

```diff
@@ -197,6 +197,15 @@ \subsubsection{Dealing with Impediments in a SoS}
 
 Particular attention should be paid to the SoS Retrospective in which the teams' representatives share any learnings or process improvements that their individual teams have succeeded with, in order to share best practices across the teams within the SoS.  
 
+\section{Reference Model}
```

Comment:
I disagree with changing the guide to reflect the training. The training has a didactic structure, the guide is intended to for consistency and lookup. For that reason I'd reject this commit, even though the renaming might have merit, didn't look further into that.